### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by yangwill**
Finally updating the main README to have simpler installation instructions and moving the configuration specific instructions to a separate file.

- - -
This change is [Reviewable](https://reviewable.io/reviews/DAIRLab/dairlib/360)

### CODE REVIEW

**Comment by yangwill** [comment]
Builds off of #359, so waiting for that PR to succeed first

**Review by Brian-Acosta — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/DAIRLab/dairlib/360)* status: 0 of 88 files reviewed, 1 unresolved discussion (waiting on @mposa and @yangwill)

___
*[`README.md` line 44 at r2](https://reviewable.io/reviews/DAIRLab/dairlib/360#-Nyf6ehJ61sEF70F-pvC:-Nyf6ehKARYR4gsquMJT:b-bl097k) ([raw file](https://github.com/DAIRLab/dairlib/blob/3673c977855ff661441f2b41b5c13c4e2cbe004c/README.md#L44)):*
> ```Markdown
> 
> ### Notes for macOS
> 1. Be sure to have Xcode 9.0 or later installed with Command Line Tools. If you receive a `clang: error: cannot specify -o when generating multiple output files` message during the build process, re-run `install_prereqs.sh`, and be sure that it runs fully before termination, as this will reconfigure Xcode to work with Drake.
> ```

I think this is pretty out of date. I would just link to the drake mac install instructions here, with a note that we don't test builds on mac, so ymmv. Maybe request that changes to build on mac get a PR.

File: examples/Cassie/osc/high_level_command.cc
Comment (Brian-Acosta) [inline examples/Cassie/osc/high_level_command.cc:1]: This update is basically free, let's not make it forced b/c we might want to use this system outside of LcmDrivenLoop

File: install/README.md
Comment (Brian-Acosta) [inline install/README.md:1]: If we are going to keep ROS Noetic instructions and infrastructure, we should tell users to revert their drake version to one that  supports Ubuntu 20.04

**Review by Brian-Acosta — COMMENTED** [review]
A few nits otherwise LGTM

**Review by Brian-Acosta — APPROVED** [review]

**Pull request merged**